modules/godot_physics_3d/godot_body_pair_3d.cpp

```diff
@@ -250,12 +250,40 @@ bool GodotBodyPair3D::_test_ccd(real_t p_step, GodotBody3D *p_A, int p_shape_A,
 	return true;
 }
 
-real_t combine_bounce(GodotBody3D *A, GodotBody3D *B) {
-	return CLAMP(A->get_bounce() + B->get_bounce(), 0, 1);
+real_t combine_bounce(GodotBody3D *A, GodotBody3D *B, int p_shapeA, int p_shapeB) {
```

Comment:
I know this is pre-existing code, but we may want to change this to follow our coding style.

```suggestion
real_t combine_bounce(GodotBody3D *p_body_a, GodotBody3D *p_body_b, int p_shape_a, int p_shape_b) {
```